superset/charts/filters.py

```diff
@@ -44,6 +45,17 @@ def apply(self, query: Query, value: Any) -> Query:
         )
 
 
+class ChartFavoriteFilter(BaseFavoriteFilter):  # pylint: disable=too-few-public-methods
+    """
+    Custom filter for the GET list that filters all charts that a user has favored
+    """
+
+    name = _("Is favorite")
+    arg_name = "chart_is_fav"
```

Comment:
Could we change this from `chart_is_fav`, `dashboard_is_fav` to `favorited` for all of them? More consistent, no abbreviation to remember, etc.

Reply:
I can't these are global

Reply:
That's an unfortunate limitation.

Reply:
Agree, I'll fix that upstream on FAB